cdap-common/bin/functions.sh

```diff
@@ -748,6 +748,9 @@ cdap_sdk_start() {
   CLASSPATH=$(find "${CDAP_HOME}/lib" -type f | sort | tr '\n' ':')
   CLASSPATH="${CLASSPATH}:${CDAP_HOME}/conf/"
 
+  # SDK requires us to be in CDAP_HOME
+  cd $(cdap_home)
```

Comment:
OK. That makes it less confusing, as it otherwise suggests that the variable was no longer valid.